MVVM Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```diff
@@ -281,6 +281,42 @@ final class FeedUIIntegrationTests: XCTestCase {
 		wait(for: [exp], timeout: 1.0)
 	}
 	
+	func test_loadFeedError_displaysErrorViewWithErrorMessage() {
+		let (sut, loader) = makeSUT()
+
+		sut.loadViewIfNeeded()
+		loader.completeFeedLoadingWithError()
+		
+		XCTAssertEqual(sut.errorMessage, localized("FEED_VIEW_CONNECTION_ERROR"))
```

Comment:
Missing pre-condition assertion (error should be nil when the view loads):

```suggestion
		sut.loadViewIfNeeded()
		XCTAssertNil(sut.errorMessage)
				
		loader.completeFeedLoadingWithError()
		XCTAssertEqual(sut.errorMessage, localized("FEED_VIEW_CONNECTION_ERROR"))
```